types/listr.d.ts

```diff
@@ -0,0 +1,51 @@
+// Type definitions for listr 0.13
+// Project: https://github.com/samverschueren/listr#readme
+// Definitions by: Dusan Radovanovic <https://github.com/durad>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
```

Comment:
why do we need this file with @types/ module ?

Reply:
I have answered above. Let me know if you have a better option.